### ISSUE TITLE
metacpan shows version 1.007 as current

### ISSUE THREAD

**Issue by twouters** [daoswald/Bytes-Random-Secure-Tiny#4]
https://metacpan.org/pod/Bytes::Random::Secure::Tiny

```
6 HOURS AGO
Distribution: Bytes-Random-Secure-Tiny
Module version: 1.007
```

**Comment by twouters** [comment]
see also: https://metacpan.org/diff/file?target=DAVIDO/Bytes-Random-Secure-Tiny-1.009/lib%2FBytes%2FRandom%2FSecure%2FTiny.pm&source=DAVIDO%2FBytes-Random-Secure-Tiny-1.008%2Flib%2FBytes%2FRandom%2FSecure%2FTiny.pm

**Comment by twouters** [comment]
This seems to be mostly an issue for https://github.com/FGasper/p5-Math-ProvablePrime/blob/78316218a48daf50c5557329e4f3b1221bafa0b1/Makefile.PL#L13

**Comment by FGasper** [comment]
It’s because the B::R::S::T version is still 1.007.

**Comment by daoswald** [comment]
This will be fixed in 1.010, being released now.
